gnoi/os/client_test.go

```diff
@@ -36,10 +38,20 @@ type activateTest struct {
 	wantErr bool
 }
 
+type verifyTest struct {
+	name,
+	runningVersion,
+	failMessage string
+}
```

Comment:
you are only using this once, perhaps making it inline downthere is better and more readable?

```
tests := []struct {
        name,
        runningVersion,
        failMessage string
}{
        {"Is Running", "version", ""},
        {"Previous Activation Fail", "version", "Activation fail"},
}
```